packages/dds/sequence/src/intervals/sequenceInterval.ts

```diff
@@ -264,27 +307,37 @@ export class SequenceIntervalClass implements SequenceInterval {
 	 * {@inheritDoc ISerializableInterval.serialize}
 	 */
 	public serialize(): ISerializedInterval {
-		const startPosition = this.client.localReferencePositionToPosition(this.start);
-		const endPosition = this.client.localReferencePositionToPosition(this.end);
-		const { startSide, endSide } = sidesFromStickiness(this.stickiness);
-		const serializedInterval: ISerializedInterval = {
+		return this.serializeDelta(this.properties, true, true) as ISerializedInterval;
+	}
+
+	public serializeDelta(
+		props: PropertySet | undefined,
+		includeStart: boolean,
+		includeEnd: boolean,
+	): SerializedIntervalDelta {
```

Comment:
I think the usage of this gets clearer if you go with this signature instead, and the implementation is about as readable:
```suggestion
	public serializeDelta({
		props: PropertySet | undefined,
		start: LocalReferencePosition | undefined,
		end: LocalReferencePosition | undefined,
	}): SerializedIntervalDelta {
```

Reply:
hmm. what we pass here will just be the intervals local references, so it would be:
`interval.serializeDelta(props, interval.start, undefined)`
it also opens the door to someone passing references which are not from the current interval, which isn't really supported.

i think i would prefer to keep booleans, but making it an object is probably better, so:
`interval.serializeDelta({props, includeStart: true, includeEnd:false})`

thoughts?

Reply:
i actually realized we don't support changing one endpoint via docs or code, so should always include both, or neither. updated for that.